metrics/collector.go

```diff
@@ -221,6 +230,12 @@ func (c *DefaultCollector) RequestRateLimited(method string) {
 	).Inc()
 }
 
+func (c *DefaultCollector) EOARateLimited(address string) {
+	c.eoaRateLimitedTransactions.
+		With(prometheus.Labels{"address": address}).
```

Comment:
let's not include the address as a label. I  think it's fine to just have a regular counter.

as a best practice, don't include arbitrary fields as labels otherwise prometheus will a separate metric for each address. On a high volume server, this can create a huge number of metrics.

Reply:
Good point :+1: Updated in https://github.com/onflow/flow-evm-gateway/pull/839/commits/a74c7135be4419785dda0e049b5c366d5639138b .
I used a plain `prometheus.Counter` instead of a `prometheus.CounterVec`, since we don't need any labels.